townhall/myapi/tests.py

```diff
@@ -62,6 +65,7 @@ def test_delete_volunteer(self):
 
 
 
+# OPPORTUNITIES
 
     def test_create_opportunity(self):
```

Comment:
I just noticed this am not sure what this is doing, please remove it since creating objects in tests isnt supposed to happen. `setUp `method handles all that